bedrock/settings/base.py

```diff
@@ -537,6 +537,7 @@ def get_app_name(hostname):
 MIDDLEWARE = [
     "allow_cidr.middleware.AllowCIDRMiddleware",
     "django.middleware.security.SecurityMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
Why do we need this?

Reply:
This is the way

Reply:
The new way that whitenoise is configured. It is removed from the wsgi.py file and added here. See https://whitenoise.evans.io/en/stable/django.html#enable-whitenoise